### PR TITLE
Migrate to latest azure sdk

### PR DESCRIPTION
**Pull request by francoisferrand**
- Bump arsenal 8.1.75
- Migrate tests to new azure storage sdk

Issue: CLDSRV-305

### CODE REVIEW

**Review by nicolas2bert — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -20,10 +20,10 @@
   "homepage": "https://github.com/scality/S3#readme",
   "dependencies": {
     "@hapi/joi": "^17.1.0",
-    "arsenal": "git+https://github.com/scality/Arsenal#8.1.72",
+    "@azure/storage-blob": "^12.12.0",
+    "arsenal": "git+https://github.com/scality/Arsenal#612904e6e93e1a9a82ed2035e3117f54ba2d5920",
```
Comment (ghost) [inline package.json]: Just a note not to forget to use a proper tag before merging

File: tests/functional/aws-node-sdk/test/multipleBackend/mpuParts/azurePutPart.js [hunk 2]
```diff
@@ -20,30 +20,34 @@ let bucketUtil;
...
+            uncommittedBlocks.forEach((l, index) => {
+                assert.strictEqual(l.name, getBlockId(uploadId,
+                    expectedParts[index].partnbr, expectedParts[index].subpartnbr));
+                assert.strictEqual(l.size, expectedParts[index].size.toString());
+            });
+            cb();
+        }, err => {
+            assert.equal(err, null, 'Expected success, got error ' +
+                `on call to Azure: ${err}`);
+            cb();
```
Comment (ghost) [inline tests/functional/aws-node-sdk/test/multipleBackend/mpuParts/azurePutPart.js]: Why not:
```suggestion
        }, cb);
```

.. as the error message will stringify and maybe reduce the information carried by the error object?

**Review by ghost — COMMENTED** [review]
Will approve after a green build (please ping me if needed)

**Review by ghost — APPROVED** [review]
Now it's green :slightly_smiling_face:

**Pull request merged**